locksmith/src/fulfillment/dispatcher.ts

```diff
@@ -286,4 +273,25 @@ export default class Dispatcher {
     })
     return transferCode
   }
+
+  isTransferSignedByLocksmith(
```

Comment:
I think we could make this more generic? `params` could be anything (not just a transfer code)

Reply:
For this, I don't think it can be any more generic. You need also need domain and types for the typed signature derivation which adds a couple more lines.